crates/telio-model/src/api_config.rs

```diff
@@ -222,6 +222,47 @@ where
     Ok(Some(eps))
 }
 
+/// Telio task monitor configuration
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize)]
+pub struct FeatureTaskMonitor {
+    /// The number of consecutive wait iterations that should trigger busy loop detection.
+    /// This typically occurs when erroneous code results in a wait loop without any awaits.
+    #[serde(default = "FeatureTaskMonitor::default_max_polls_immediately")]
+    pub max_polls_immediately_ms: Option<u64>,
```

Comment:
The comment talks about iterations but the naming of the field mentions time (ms), unless ms means something else in this case, in which case that should be clarified